secret.tf

```diff
@@ -0,0 +1,14 @@
+data "azurerm_key_vault" "main" {
+  count = (length(var.frontends) != 0  && var.store_privateip == "true") ? 1 : 0
```

Comment:
is there a reason `store_privateip` isn't a boolean?

Reply:
changed the code.